persistent/Database/Persist.hs

```diff
@@ -40,6 +63,26 @@ infixr 3 =., +=., -=., *=., /=.
   forall v typ.  PersistField typ => EntityField v typ -> typ -> Update v
 
 -- | Assign a field a value.
+--
+-- Example usage:
```

Comment:
I think examples are better when marked up as:
```
-- ==== __Example Usage__
```

this causes them to be made into collapsible/expandable sections in the Haddocks, so that you can have lots of examples without making scrolling through functions harder. The docs for `<$>` do this, [for example](https://hackage.haskell.org/package/base-4.8.2.0/docs/Control-Applicative.html).

![image](https://cloud.githubusercontent.com/assets/1274145/13133410/cdd601c0-d5af-11e5-9571-ddc93ad73072.png)